src/specklepy/core/api/resources/current/project_resource.py

```diff
@@ -432,3 +442,84 @@ def update_role(self, input: ProjectUpdateRoleInput) -> ProjectWithTeam:
         return self.make_request_and_parse_response(
             DataResponse[DataResponse[ProjectWithTeam]], QUERY, variables
         ).data.data
+
+    def get_projects_with_permissions(
```

Comment:
This is the same as the one in `ActiveUserResource` ? 
I would like to keep `ProjectResource` just for `Project` queries/mutations, not `activeUser.projects` based queries...